.github/actions/changeset-status/action.yml

```diff
@@ -37,16 +37,22 @@ runs:
         fi
         echo "✅ Temporary file created at: $OUTPUT"
 
-        # Run the changeset status command with relative path output
-        yarn changeset status ${{ inputs.args }} --output $(realpath --relative-to . "$OUTPUT")
+        RELATED_OUTPUT=$(realpath --relative-to . "$OUTPUT")
 
-        if [ ! -s "$OUTPUT" ]; then
-          echo "🚨 Error: Temporary file is empty after running 'yarn changeset status'"
-          exit 1
-        fi
+        # Run the changeset status command with relative path output
+        # and catch an error if the command fails
+        { # try
+          CAUGHT_OUTPUT=$(yarn changeset status ${{ inputs.args }} --output $RELATED_OUTPUT 2>&1)
+        } || { # catch
+          # If the error is NOT caused by empty changesets, print the error and exit
+          if ! echo "$CAUGHT_OUTPUT" | grep -q "Some packages have been changed but no changesets were found"; then
```

Comment:
is there other way to catch the `Some packages have been changed but no changesets were found` case?

Reply:
I bet there is, but I have no idea how to do it in bash :( 

Unfortunately, the changeset CLI does not offer additional options/flags to the command to handle that case.

@aalves08 I am wide open to suggestions if you have anything in mind 🙏